Tests/AGSnapshotHelperTests/Categories/NotificationCenter+PostContentSizeCategoryChangeTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  NotificationCenter+PostContentSizeCategoryChangeTests.swift
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 21/01/2018.
+//  Copyright © 2018 Grupa Allegro. All rights reserved.
+//
+
+import AGSnapshotHelper
```

Comment:
@testable?

Reply:
Tested method is marked as `public`, so @testable is not required.